src/main/java/seedu/address/model/item/Quantity.java

```diff
@@ -83,4 +92,8 @@ public int hashCode() {
         return value.hashCode();
     }
 
+    @Override
+    public int compareTo(Quantity o) {
+        return Double.compare(Double.parseDouble(value), Double.parseDouble(o.value));
+    }
 }
```

Comment:
Sorry this one very nitpicky, but if you're changing the "o's", can you change this one too

Reply:
Changed!